src/C-DEngine/C-DEngines/C-DThingService/cdeTheThingSensorModel.cs

```diff
@@ -198,6 +198,14 @@ public List<cdeP> GetSensorProperties()
         {
             return GetAllProperties(10).Where(p => p.IsSensor== true).ToList();
         }
+        /// <summary>
+        /// Returns all properties that are marked as Sensors.
+        /// </summary>
+        /// <returns></returns>
+        public List<cdeP> GetConfigProperties()
+        {
+            return GetAllProperties(10).Where(p => p.IsConfig == true).ToList();
```

Comment:
Can this fail if GetAllProperties(10) returns null?

Reply:
GetAllProperties(n) doesn't return null, at least right now. Added checks anyway.